substrate/frame/identity/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-identity"
-version = "29.0.1"
+version = "30.0.0"
```

Comment:
`30.0.0` is already released (guessing with sdk v1.8.0)

```suggestion
version = "29.0.2"
```

Reply:
Looks like latest on crates.io is v34.0.0, so there's some versions in between. Anyway, updated it to v29.0.2.